ocaml/xapi/dune

```diff
@@ -74,6 +74,7 @@
    rrdd-plugin.base
    rrdd-plugin.local
    sexplib
+   sexplib0
```

Comment:
Unrelated note to the PR: I wish we could just use 1 library for S-expressions instead of 3 (on top of that there's also another in xs-opam that dune uses: csexp)

Reply:
why have n sexp libraries when you can have n+1?